ProcessMaker/processmaker
# Create Default Templates

ticket [FOUR-7770](https://processmaker.atlassian.net/browse/FOUR-7770)


## Issue & Reproduction Steps
As a Project Manager, I want to access templates by default in my environment so that I can get my process live more quickly and see more use cases for how I can build my process

## Solution
- Create seeders for the default templates and default category

## How to Test
- Please rollback to create_process_templates_table. eg: `php artisan migrate:rollback --step=2`
- Run `php artisan migrate`
- Make sure you have all the packages installed
- Click on the designer top menu
- Click on the Templates tab
- Select one of the default templates (Employee Onboarding 2023)
- Run the process and make sure it works as expected
-- To make sure the process is working as expected, please make sure you have the setup the default email server
-- For the last step of the process, Create User Script you may need to follow this steps -> ([Script Task cURL Error](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/2820603905/Error+Knowledge+Base#Script-Task-cURL-Error))
-- The last step of the process is handled by a timer so locally, you may need to run these commands.
-- `php artisan schedule:list`
-- `php artisan schedule:work`
-- `php artisan bpmn:timer`




## Code Review Checklist


[FOUR-7770]: https://processmaker.atlassian.net/browse/FOUR-7770?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ci:db:clean
.........
